app/helpers/simple_server_env_helper.rb

```diff
@@ -0,0 +1,26 @@
+module SimpleServerEnvHelper
+  SIMPLE_SERVER_ENV = "SIMPLE_SERVER_ENV".freeze
```

Comment:
This is not really necessary; just put the string in the `ENV.fetch()` lines below. If we have to change this ever, we'll have to grep the entire codebase anyway.

Reply:
Replaced variable with string in the `ENV.fetch` call itself.